lib/iris/tests/test_netcdf.py

```diff
@@ -147,6 +147,14 @@ def test_load_tmerc_grid_with_projection_origin(self):
         self.assertEqual(cube.coord('projection_y_coordinate').coord_system,
                          expected)
 
+    def test_load_laea_grid(self):
+        # Test loading a single CF-netCDF file with a lambert azimuthal equal
+        # area grid_mapping.
+        cube = iris.load_cube(
+            tests.get_data_path(('NetCDF', 'lambert_azimuthal_equal_area',
+                                 'euro_air_temp.nc')))
+        self.assertCML(cube, ('netcdf', 'netcdf_laea.cml'))
```

Comment:
(As an aside, this test is only needed because it's almost impossible to unit test the Pyke rules - it'd be good do something about that, but not in this PR. cc: @bjlittle)